dango/dex/src/execute.rs

```diff
@@ -517,6 +523,10 @@ fn clear_orders_of_pair(
         volume,
     })?;
 
+    let dango_config = querier.query_dango_config()?;
+    let oracle = dango_config.addresses.oracle;
+    let account_factory = dango_config.addresses.account_factory;
```

Comment:
You're querying these for every trading pair. Better to query once and pass the `dango_config` as a input to the `clear_orders_of_pair` function.

Reply:
See: https://github.com/left-curve/left-curve/pull/546/commits/22186862ef7fee317d7555a6692a6f07ae37166a